.github/pull_request_template.md

```diff
@@ -12,7 +12,9 @@
 
 ## Change checklist
 
-- [ ] Self-review.
-- [ ] Documentation updates following the [style guide](https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#appendix-a-full-conventions-text), if relevant.
-- [ ] Tests if relevant.
-- [ ] All breaking changes documented.
+Please make sure to consider all of these:
+
+- Self-review.
+- Documentation updates following the [style guide](https://rust-lang.github.io/rfcs/1574-more-api-documentation-conventions.html#appendix-a-full-conventions-text).
+- Tests.
+- Document breaking changes.
```

Comment:
maybe put them behind html comments, so they don't show up in the actual issue?

Reply:
Is that desired though?  Then folks will just not see this at all.

I get that it could be nice to hide them in the commit messages, but the html comments do not achieve that I think.